src/Sylius/Behat/Service/Mocker/PaypalApiMockerInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Service\Mocker;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+interface PaypalApiMockerInterface
```

Comment:
I guess that you should remove this commit: https://github.com/Sylius/Sylius/pull/5415/commits/9cf224587032a91d83df375bd7efe43d18d387de, it changes nothing to the current implementation of thank you page